### PR TITLE
Add recipe for mindre-theme

### PR DESCRIPTION
**Pull request by erikbackman**
### Brief summary of what the package does

Mindre (which is the Swedish word for “less”) tries to strike a good balance
between usability and minimalism by almost being a monochrome theme but with
a splash of color.

### Direct link to the package repository

https://github.com/erikbackman/mindre-theme

### Your association with the package

Maintainer

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [ ] I have confirmed some of these without doing them

### CODE REVIEW

File: recipes/mindre-theme [hunk 1]
```diff
@@ -0,0 +1 @@
+(mindre-theme :fetcher github :repo "erikbackman/mindre-theme" :files ("mindre-theme.el"))
```
Comment (riscy) [inline recipes/mindre-theme:1]: Prefer equivalent recipe unless there's a good reason not to:
```suggestion
(mindre-theme :repo "erikbackman/mindre-theme" :fetcher github)`
```

**Review by riscy — COMMENTED** [review]
Thanks for this, looks like a very nice theme.

Re: mindre-theme.el#L3: Have you done the paperwork to assign this copyright?  I check because this is a common copy/paste error.  And if you *have* done the paperwork, it seems like it would make sense to host this on ELPA as a first step.

**Comment by erikbackman** [comment]
> Re: mindre-theme.el#L3: Have you done the paperwork to assign this copyright? I check because this is a common copy/paste error. And if you have done the paperwork, it seems like it would make sense to host this on ELPA as a first step.

I have not. It's a copy/paste error.

**Comment by riscy** [comment]
This is a little bit pedantic, but the `;; COPYRIGHT NOTICE` heading at the top is actually over the licensing terms, which are a different animal.  You could safely just remove that whole line.

Everything looks good though - happy to merge.

**Pull request merged**